website/react-magma-docs/src/pages/data-visualization-intro/introduction.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Getting Started
+order: 1
+---
+
+import { LeadParagraph } from '../../components/LeadParagraph';
+import { Flex, FlexBehavior, FlexAlignItems, magma } from 'react-magma-dom';
+
+<PageContent componentName="getting_started" type="data_visualization_intro">
```

Comment:
We can remove https://docs-preview-1544--upbeat-sinoussi-f675aa.netlify.app/api/chart/ now that this page exists